state/state.go

```diff
@@ -187,9 +198,11 @@ func (s *State) loadValidators(height int) *ValidatorsInfo {
 	wire.ReadBinaryPtr(v, r, 0, n, err)
 	if *err != nil {
 		// DATA HAS BEEN CORRUPTED OR THE SPEC HAS CHANGED
-		cmn.Exit(cmn.Fmt("LoadValidators: Data has been corrupted or its spec has changed: %v\n", *err))
+		cmn.Exit(cmn.Fmt(`LoadValidators: Data has been corrupted or its spec has changed:
```

Comment:
Perhaps this is a little nitty about " vs `, but that's alright :)

Reply:
I'd like to use " instead of `, but that means that I can't line wrap it after 99 characters. :-)

" doesn't support line breaks and I'm getting more and more used to split screens and multiple windows and having line breaks at 100 is great.